docs/rsdl/rapid-rsdl-abnf.md

```diff
@@ -46,7 +47,23 @@ structuredType       = annotations [ %s"abstract" RWS ] %s"type" RWS identifier
 
 structuredTypeMember = property / operation ; property, action, or function
 
-property             = annotations [propertyModifier RWS] identifier OWS ":" OWS typeReference
+property             = structuralProperty / referenceProperty
```

Comment:
This is "semantic ABNF" because a parser can't know which kind it has encountered.